app/src/pages/inside/logsPage/tabs/attachments/attachments.jsx

```diff
@@ -0,0 +1,70 @@
+// @ts-check
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Carousel } from 'react-responsive-carousel';
+import { connect } from 'react-redux';
+import { getIcon } from 'common/img/launch/attachments';
+import { attachmentsSelector } from 'controllers/attachments';
+import 'react-responsive-carousel/lib/styles/carousel.css';
+import styles from './attachments.scss';
+
+const cx = classNames.bind(styles);
+
+@connect((state) => ({
+  logItems: attachmentsSelector(state),
```

Comment:
Is it actually array of Log Items or Array of Attachments?

Reply:
Good point. It's the attachments!